js/app.js

```diff
@@ -1 +1,116 @@
-console.log('Hello frontend');
+document.addEventListener("DOMContentLoaded", function(){
+
+  var playerXTurn = true;
+  var board = document.getElementById("board");
+  var resetButton = document.getElementById("reset");
+  var space;
+
+  addEventListener();
+
+  function turnTrackerAlert(){
+    if (playerXTurn == false){
+      document.getElementById("turn").innerHTML="Your turn O!";
+    } else{
+      document.getElementById("turn").innerHTML="Your turn X!";
+    }
+  };
+
```

Comment:
I like that you chose "innerHTML" as an approach to switch in and out "X", "O", and "empty". This will allow you to quickly change the style this hypertext in the future, if desired :)